src/pages/builders/index.ts

```diff
@@ -277,59 +317,70 @@ export class CypressScriptBuilder extends ScriptBuilder {
   click = (selector: string, causesNavigation: boolean) => {
     this.pushCodes(`cy.get('${selector}').click()`);
     return this;
-  }
+  };
 
   hover = (selector: string, causesNavigation: boolean) => {
     this.pushCodes(`cy.get('${selector}').trigger('mouseover')`);
     return this;
-  }
+  };
 
   load = (url: string) => {
     this.pushCodes(`cy.visit('${url}')`);
     return this;
-  }
+  };
 
   resize = (width: number, height: number) => {
     this.pushCodes(`cy.viewport(${width}, ${height})`);
     return this;
-  }
+  };
 
   fill = (selector: string, value: string, causesNavigation: boolean) => {
     this.pushCodes(`cy.get('${selector}').type('${value}')`);
     return this;
-  }
+  };
 
   type = (selector: string, value: string, causesNavigation: boolean) => {
     this.pushCodes(`cy.get('${selector}').type('${value}')`);
     return this;
-  }
+  };
 
   select = (selector: string, option: string, causesNavigation: boolean) => {
     this.pushCodes(`cy.get('${selector}').select('${option}')`);
     return this;
-  }
+  };
 
   keydown = (selector: string, key: string, causesNavigation: boolean) => {
     this.pushCodes(`cy.get('${selector}').type('{${key}}')`);
     return this;
-  }
+  };
 
-  wheel = (deltaX: number, deltaY: number) => {
-    this.pushCodes(`cy.scrollTo(${Math.floor(deltaX)}, ${Math.floor(deltaY)})`);
+  wheel = (
+    deltaX: number,
+    deltaY: number,
+    pageXOffset?: number,
+    pageYOffset?: number
+  ) => {
+    this.pushCodes(
+      `cy.scrollTo(${Math.floor(pageXOffset ?? 0)}, ${Math.floor(
+        pageYOffset ?? 0
+      )})`
+    );
     return this;
-  }
+  };
 
   fullScreenshot = () => {
-    this.pushCodes(`cy.screenshot('screenshot.png')`);
+    this.pushCodes(`cy.screenshot()`);
```

Comment:
Cypress screenshots don't require a file name, and if we were to provide one it'd have to be just `screenshot` without file extension